homeassistant/components/device_tracker/__init__.py

```diff
@@ -67,8 +67,13 @@
 ATTR_HOST_NAME = 'host_name'
 ATTR_LOCATION_NAME = 'location_name'
 ATTR_GPS = 'gps'
+ATTR_GPS_UPDATED = 'gps_updated'
```

Comment:
Lets remove gps_updated completely. With the new `source_type` And setting the coords to home gps_update is really redundant, since last_updated is now always valid